services/brig/src/Brig/API/Error.hs

```diff
@@ -191,6 +191,11 @@ deleteUserError DeleteUserInvalidPassword = StdError (errorToWai @'E.BadCredenti
 deleteUserError DeleteUserMissingPassword = StdError (errorToWai @'E.MissingAuth)
 deleteUserError (DeleteUserPendingCode t) = RichError deletionCodePending (DeletionCodeTimeout t) []
 deleteUserError DeleteUserOwnerDeletingSelf = StdError (errorToWai @'E.OwnerDeletingSelf)
+deleteUserError (DeleteUserVerificationCodeThrottled t) =
```

Comment:
What about the other places that require a 2fa code?  Are they all covered by `verificationCodeThrottledError`?  why is `deleteUserError` special?  (I feel I should know this...)

Reply:
not important.  the important thing is that all the UI flows that rely on table `vcodes` are covered by the changes here, and that's the case.